Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -204,6 +218,9 @@ handler transporter req quote = do
       otp <- generateOTPCode
       now <- getCurrentTime
       trackingUrl <- buildTrackingUrl guid
+      id1 <- Id <$> generateGUID
+      let fromLocation = booking.fromLocation{id = id1}
+      toLocation <- mapM generateUniqueLocationId booking.toLocation
```

Comment:
Why this?

Reply:
because at BPP we assign same search request location to Booking, this location with ids already exists in Location table, when creating booking it throws duplicate key errors. thats why we are updating the location ids for booking